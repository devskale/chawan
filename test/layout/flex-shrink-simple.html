<!DOCTYPE html>
<div style="display: flex">
<div>test test test test test test test test test</div>
<div>test test test test test test test test test test test test test test test test test</div>
<div>last</div>
