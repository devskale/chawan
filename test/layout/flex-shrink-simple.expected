test test test test test  test test test test test test test test test test last
test test test test       test test test test test test test
